ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/functions/[slug]/logs/NewReplayModal.tsx

```diff
@@ -248,11 +250,13 @@ export default function NewReplayModal({ functionSlug, isOpen, onClose }: NewRep
                 <p aria-label={`Number of ${label} runs`} className="text-sm text-slate-500">
                   {isLoading ? (
                     <Placeholder className="top-px inline-flex h-3 w-3 bg-slate-200" />
-                  ) : (
+                  ) : isTimeRangeValid ? (
                     count.toLocaleString(undefined, {
                       notation: 'compact',
                       compactDisplay: 'short',
                     })
+                  ) : (
+                    '-'
```

Comment:
From @anafilipadealmeida's feedback, can we make this blank? Maybe using a nbsp to prevent layout shift?